index.html

```diff
@@ -97,11 +97,17 @@ <h1 class="multi-post-title">Multi-Post Stories</h1>
             <li class="multi-btn">Bootstrap</li>
             <li class="multi-btn">Ruby</li>
           </ul>
-          <button class="see-project-btn for-hover for-pressed" type="button">See Project</button>
+          <button
+            class="see-project-btn for-hover for-pressed click-popup"
+            data-position="0"
+            type="button"
+          >
+            See Project
+          </button>
         </div>
       </div>
       <div class="six-wrap">
```

Comment:
- As shown in the project requirement below, all of your works card should be generated dynmaically using JavaScript rather than throgh html like you are currently doing. To achieve this you would make use of the projects array you have created and dynamically loop through its contents to display them on the webpage when the page loads. 
- [ ] I suggest that you kindly ensure the buttons in the popup are links which can be visited to view the project's live version or github repository
- Please make sure to make this change and good job on creating the popup correctly. :+1: :sparkles: 
- > project requirement
![nm](https://github.com/mryeminthu/Portfolio/assets/89055502/8e631956-9377-4e7c-8225-dd3d180b166b)